pkg/contract-manager/cr_services.go

```diff
@@ -0,0 +1,150 @@
+package contractmanager
+
+import (
+	"context"
+
+	nodecorev1alpha1 "fluidos.eu/node/api/nodecore/v1alpha1"
+	reservationv1alpha1 "fluidos.eu/node/api/reservation/v1alpha1"
+	discoverymanager "fluidos.eu/node/pkg/discovery-manager"
+	localResourceManager "fluidos.eu/node/pkg/local-resource-manager"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/klog"
+)
+
+// getAllFlavoursCRsSpec returns all the Flavour CRs Spec in the cluster
+func getAllFlavoursCRsSpec() ([]nodecorev1alpha1.FlavourSpec, error) {
+	cl, err := localResourceManager.GetKClient(context.Background())
+	utilruntime.Must(err)
+
+	var flavourList nodecorev1alpha1.FlavourList
+
+	// List all Flavour CRs
+	err = cl.List(context.Background(), &flavourList)
+	if err != nil {
+		return nil, err
+	}
+
+	var flavoursSpec []nodecorev1alpha1.FlavourSpec
+
+	// Get the Spec of each Flavour
+	for _, flavour := range flavourList.Items {
+		flavoursSpec = append(flavoursSpec, flavour.Spec)
+	}
+
+	return flavoursSpec, nil
+}
+
+// getAllFlavoursCRs returns all the Flavour CRs in the cluster
+func getAllFlavoursCRs() ([]nodecorev1alpha1.Flavour, error) {
+	cl, err := localResourceManager.GetKClient(context.Background())
+	utilruntime.Must(err)
+
+	var flavourList nodecorev1alpha1.FlavourList
+
+	// List all Flavour CRs
+	err = cl.List(context.Background(), &flavourList)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
This should work
```suggestion
return flavourList.Items
```

Reply:
It was one of the old comment, fixed (see conversation on `pkg/contract-manager/cr_services.go`)